app/src/main/res/values/strings.xml

```diff
@@ -281,8 +281,8 @@
   <string name="upload_problem_image">Problems found in image</string>
   <string name="internet_downloaded">Please only upload pictures that you have taken by yourself. Don\'t upload pictures that you have downloaded from the Internet.</string>
   <string name="give_permission">Give permission</string>
-  <string name="use_external_storage">Use external storage</string>
-  <string name="use_external_storage_summary">Save pictures taken with the in-app camera on your device</string>
+  <string name="use_external_storage">Save pictures</string>
+  <string name="use_external_storage_summary">Save pictures taken with the in-app camera to your device's disk</string>
```

Comment:
I'm a little iffy about this. How about using "storage" instead of "disk" here? I feel "storage" is a more commonly term used in Android when compared to "disk".

Reply:
Also, it looks like the apostrophe has to be escaped here. So, you would've to use `&apos;` instead of `'`

Reply:
```suggestion
  <string name="use_external_storage_summary">Save pictures taken with the in-app camera to your device storage</string>
```